examples/cpu-count/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cpu-count",
+  "version": "0.1.0",
+  "description": "Neon CPU Count",
+  "main": "index.node",
+  "scripts": {
+    "build": "cargo-cp-artifact -nc index.node -- cargo build --message-format=json-render-diagnostics",
+    "install": "npm run build",
+    "test": "cargo test"
+  },
+  "author": "K.J. Valencik",
```

Comment:
Its probably better to attribute to neon contributors as opposed to single author

Reply:
I don't feel strongly. It's getting pulled in by the default tooling (`npm init`). I'd just assume delete the key.